test/api_integration/apis/core/index.js

```diff
@@ -33,6 +33,12 @@ export default function({ getService }) {
             200,
             'SavedObjects client: {"page":1,"per_page":20,"total":0,"saved_objects":[]}'
           ));
+
+      it('provides access to application rendering client', async () => {
+        await supertest.get('/requestcontext/render').expect(200, /app:core/);
+        await supertest.get('/requestcontext/render/core').expect(200, /app:core/);
+        await supertest.get('/requestcontext/render/testbed').expect(200, /app:testbed/);
```

Comment:
What it asserts? as I can see the id ignored in https://github.com/elastic/kibana/pull/52161/files#diff-53e97c5775924078ca523da2167f8b72R90